data/manager.php

```diff
@@ -4,6 +4,7 @@
 
```

Comment:
Move all core data to use V2.

Reply:
Handled https://github.com/elementor/elementor/pull/14205/commits/f60bf5710ddfb9ae44b0e95271c9427ee566d01a

Reply:
https://github.com/elementor/elementor/pull/14205/commits/6aba6f78bd29408f5e1f7bb44e25dbaedb1f3c2a

Reply:
https://github.com/elementor/elementor/pull/14205/commits/0e3a49b332c3af4711be0fda721a98eab85b6113